packages/kit/src/runtime/client/client.js

```diff
@@ -1513,6 +1528,54 @@ export function create_client({ target, base }) {
 				});
 			});
 
+			target.addEventListener('submit', (event) => {
+				if (event.defaultPrevented) return;
+
+				const form = /** @type {HTMLFormElement} */ (
+					HTMLFormElement.prototype.cloneNode.call(event.target)
+				);
+
+				const submitter = /** @type {HTMLButtonElement | HTMLInputElement} */ (event.submitter);
+
+				const method = event.submitter?.hasAttribute('formmethod')
+					? submitter.formMethod
+					: form.method;
+
+				if (method !== 'get') return;
+
+				const url = new URL(
+					event.submitter?.hasAttribute('formaction') ? submitter.formAction : form.action
```

Comment:
```suggestion
					submitter?.formAction ?? form.action
```

Reply:
Sadly we can't do this for the reason given above

Reply:
Won't work, see the code inside `runtime/app/forms.js` (the web platform is bonkers)